### PR TITLE
Enabled bf16 atomic_add

### PR DESCRIPTION
**Pull request by zjing14**
Enabled bf16 atomic_add on MI300

### CODE REVIEW

**Review by aska-0096 — DISMISSED** [review]
Looks good to me

**Comment by aska-0096** [comment]
> Looks good to me

We can merge it.
@junliume

File: profiler/src/profile_gemm_multiply_multiply.cpp [hunk 1]
```diff
@@ -76,11 +77,13 @@ int profile_gemm_multiply_multiply(int argc, char* argv[])
     int n_warmup      = 1;
     int n_iter        = 10;
     uint64_t rotating = 0;
-    if(argc == 19)
+    int KBatch        = 1;
```
Comment (aska-0096) [inline profiler/src/profile_gemm_multiply_multiply.cpp:80]: We can iterate KBatchList when KBatch input is <0

**Comment by illsilin** [comment]
Hey Jing, just a casual reminder to please run the clang-format script.

**Comment by zjing14** [comment]
> profiler/src/profile_gemm_multiply_multiply.cpp

Looks like CK only accepts clang-format-12.

**Comment by zjing14** [comment]
> Hey Jing, just a casual reminder to please run the clang-format script.

Looks like CK only accepts `clang-format-12`. Is any way to do clang-format with `clang-format`?

**Comment by zjing14** [comment]
@aska-0096 looks like there are failures with MI200. Could you take a look? I have no MI200 access.

**Comment by illsilin** [comment]
Hi Jing, you can just install the clang-format-12 with:
apt-get update
apt-get install clang-format-12

File: profiler/src/profile_gemm_multiply_multiply.cpp (hunk 1 above)
Comment (zjing14) [reply profiler/src/profile_gemm_multiply_multiply.cpp:80]: Done

**Comment by zjing14** [comment]
> Hi Jing, you can just install the clang-format-12 with: apt-get update apt-get install clang-format-12

Yes, I know that. The problem I cannot do that on my side.

**Pull request closed without merging**